app/models/comment.rb

```diff
@@ -1,4 +1,14 @@
 class Comment < ApplicationRecord
   belongs_to :user, foreign_key: 'author_id'
   belongs_to :post
+
+  alias_attribute :author, :user
+
+  after_create :update_post_comments_counter
+
+  private
+
+  def update_post_comments_counter
+    post.update_comments_counter
+  end
```

Comment:
- [ ] Kindly add `unit specs` for this `update_post_comments_counter` method as well :+1: